src/dial.rs

```diff
@@ -80,18 +80,10 @@ pub struct DialAlarm {
     pub dial_id: usize,
     pub time: Instant,
     pub correct_key: char,
+    pub sound_id: u64,
```

Comment:
Good idea, I'm not sure why I didn't just do this. The only issue is that nothing uses it, it still uses the get_id() method which could be removed and replaced with just an access to this field.

Reply:
Will do, looks like the best option is to store it like you said